cogs/scheduledmessage.py

```diff
@@ -0,0 +1,24 @@
+from discord.ext import tasks, commands
+
+class ScheduledMessage(commands.Cog):
+    def __init__(self, bot: commands.Bot, message: str, channelID: int, intervalSeconds: int):
+        self.bot = bot
+        self.message = message
+        self.channelID = channelID
+        self.intervalSeconds = intervalSeconds
+        self.pass_interval(self.intervalSeconds)
```

Comment:
I would make `printer()` and `before_printer()` public methods and call `printer.start()` here.

Reply:
Attempted a fix by decorating in __init__.